Lib/test/test_sys_settrace.py

```diff
@@ -346,7 +346,7 @@ def make_tracer():
         return Tracer()
 
     def compare_events(self, line_offset, events, expected_events):
-        events = [(l - line_offset, e) for (l, e) in events]
+        events = [(l - line_offset if l is not None else None, e) for (l, e) in events]
```

Comment:
I introduced for debugging this PR. It is no longer strictly necessary, but it gives nicer output when tests fail.